core/templates/pages/technical-feedback-dashboard-page/technical-feedback-dashboard-page.component.html

```
@@ -0,0 +1,28 @@
+    <button
+      class="btn btn-primary"
+      (click)="fetchListButton()">
+      Get List of LEAP Reports
+    </button>
+    <button
+      class="btn btn-primary"
+      (click)="getDetailedViewButton()">
+      view LEAP report
+    </button>
+    <input
+      type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)